Cassiopee/Generator/Generator/IBM.py

```diff
@@ -1082,6 +1081,56 @@ def buildOctree(tb, dimPb=3, vmin=15, snears=0.01, snearFactor=1., dfars=10., df
 #==============================================================================
 # 
 #==============================================================================
+def createRefinementBodies(tb, dimPb=3, hmod=0.01):
+    """Creates refinement bodies from the immersed boundaries to extend the finest resolution in the fluid domain."""
+    import Geom.IBM as D_IBM
+    import Geom.Offset as O
+
+    pointsPerUnitLength = 10 if dimPb == 3 else 100
```

Comment:
would it not be better to let the use decide the pointsPerUnitLength and if they are not sure just have a default value?

Reply:
I don't think so. An offset of a single part of the whole geometry is more sensitive. I'd rather have a fixed parameter of our own that can be changed in the future.

Reply:
Ok, so we are currently deeming that the values provided are good for all foreseeable cases. If in the future we have a case for which they are not suitable, we will change the values as needed and recompile?

Reply:
That's the spirit. These values have been tested for the multielement airfoil in 2D and the HL-CRM in 3D. If anyone finds more generalized values, I'd be than happy to change the default parameters!